Sources/PackageExtension/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageExtension
```

Comment:
This glue is all fairly similar to the corresponding logic for PackageDescription, except that it doesn't have to build two different versions.